packages/wp-now/src/run-cli.ts

```diff
@@ -0,0 +1,78 @@
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { startServer } from './start-server';
+import { portFinder } from './port-finder';
+import { DEFAULT_PHP_VERSION } from './constants';
+import { SupportedPHPVersion } from '@php-wasm/universal';
+
+function startSpinner(message: string) {
+	process.stdout.write(`${message}...\n`);
+	return {
+		succeed: (text: string) => {
+			console.log(`${text}`);
+		},
+		fail: (text: string) => {
+			console.error(`${text}`);
+		},
+	};
+}
+
+export async function runCli() {
+	const port = await portFinder.getOpenPort();
+	return yargs(hideBin(process.argv))
+		.scriptName('wp-now')
+		.usage('$0 <cmd> [args]')
+		.command(
+			'start',
+			'Start the server',
+			(yargs) => {
+				yargs.option('port', {
+					describe: 'Server port',
+					type: 'number',
+					default: port,
+				});
+				yargs.option('path', {
+					describe:
+						'Path to the PHP or WordPress project. Defaults to the current working directory.',
+					type: 'string',
+					default: process.cwd(),
+				});
+				yargs.option('wp-content', {
+					describe:
+						'Path to the wp-content directory. Defaults to .wp-now/projects/ inside the home directory.',
+					type: 'string',
+					default: '',
+				});
+				yargs.option('phpVersion', {
```

Comment:
Can we rename this argument to `php` for simplicity?

Reply:
Renamed here: https://github.com/WordPress/wordpress-playground/pull/188/commits/bc0573d2049b2f1f61ae5b83b6e6c4bd0bfd100e